### PR TITLE
Update ELN-0603 - Registry for Identifiers.md

### PR DESCRIPTION
**Pull request by Razumain**
OID Registry updates

### CODE REVIEW

File: ELN-0603 - Registry for Identifiers.md [hunk 1]
```diff
@@ -420,48 +420,50 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
 <a name="asn1-declarations"></a>
 #### 3.2.1. ASN.1 Declarations
 
-    -- Object Identifier Registry for the Swedish E-identification board
+Object Identifier Registry for Sweden Connect OID registry <sup>1</sup>
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: "Object Identifier Registry for Sweden Connect" is sufficient...

File: ELN-0603 - Registry for Identifiers.md [hunk 2]
```diff
@@ -420,48 +420,50 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
...
 
     id-eleg OBJECT IDENTIFIER ::= {iso(1) member-body(2) se(752) e-legitimationsnamnden(201)}
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: Is e-legitimationsnamnden registered or can we change to sweden-connect(201)?

File: ELN-0603 - Registry for Identifiers.md [hunk 3]
```diff
@@ -420,48 +420,50 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
...
     id-mod    OBJECT IDENTIFIER ::= { id-eleg 0 }    -- ASN.1 modules
     id-test   OBJECT IDENTIFIER ::= { id-eleg 1 }    -- OIDs for test
     id-pol    OBJECT IDENTIFIER ::= { id-eleg 2 }    -- Policy
     id-attr   OBJECT IDENTIFIER ::= { id-eleg 3 }    -- Attributes
     id-qcs    OBJECT IDENTIFIER ::= { id-eleg 4 }    -- QC Statement
     id-ce     OBJECT IDENTIFIER ::= { id-eleg 5 }    -- Cert Extensions
 
-    -- E-legnamnden modules
+    -- Sweden Connect modules
+    id-mod-auth-context-88 OBJECT IDENTIFIER ::= { id-mod 1 } - Used in RFC 7773
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: Double hyphens should be used.

File: ELN-0603 - Registry for Identifiers.md [hunk 4]
```diff
@@ -420,48 +420,50 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
...
-    id-attr-eidas-np-address     OBJECT IDENTIFIER ::= { id-attr 9 }    -- eIDAS Natural Person Address    
-    id-attr-user-certificate     OBJECT IDENTIFIER ::= { id-attr 10 }   -- User certificate    
-    id-attr-user-signature       OBJECT IDENTIFIER ::= { id-attr 11 }   -- User signature    
+    id-attr-birth-name           OBJECT IDENTIFIER ::= { id-attr 8 }    -- Birth name
+    id-attr-eidas-np-address     OBJECT IDENTIFIER ::= { id-attr 9 }    -- eIDAS Natural Person Address
+    id-attr-user-certificate     OBJECT IDENTIFIER ::= { id-attr 10 }   -- User certificate
+    id-attr-user-signature       OBJECT IDENTIFIER ::= { id-attr 11 }   -- User signature
     id-attr-sad                  OBJECT IDENTIFIER ::= { id-attr 12 }   -- Signature activation data
     id-attr-auth-srv-signature   OBJECT IDENTIFIER ::= { id-attr 13 }   -- Authentication server signature
-    id-attr-sign-message-digest  OBJECT IDENTIFIER ::= { id-attr 14 }   -- Sign message digest
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: Why did you remove the signMessageDigest attribute?

File: ELN-0603 - Registry for Identifiers.md [hunk 5]
```diff
@@ -420,48 +420,50 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
...
-    id-qcs-statement   OBJECT IDENTIFIER ::= { id-qcs 2 }   -- QC statements
+    id-qcs-statement   OBJECT IDENTIFIER ::= { id-qcs 2 }   –- QC statements
+
+    -- Sweden Connect Certificate Extensions
+    id-ce-authContext  OBJECT IDENTIFIER ::= { id-ce 1 }    -- Auth context extension used in RFC 7773
+    id-ce-svt          OBJECT IDENTIFIER ::= { id-ce 2 }    -- Signature Validation Token extension
 
-    -- E-legnamnden Certificate Extensions
-    id-ce-authContext  OBJECT IDENTIFIER ::= { id-ce 1 }
+> 1. The OID 1.2.752.201 was assigned to the Swedish E-identification board. This organisation is overtaken by the Swedish Agency for Digital Government who uses this OID arc for the Sweden Connect technical framework.
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: Detail. Change to:
> \[1\]: The OID 1.2.752.201 ...

File: ELN-0603 - Registry for Identifiers.md [hunk 6]
```diff
@@ -546,6 +548,8 @@ The following OIDs are defined in the ASN.1 declarations in [3.2.1](#asn1-declar
 
 - The Sign Message Authentication Context URIs defined in section 3.1.1.1 are deprecated.
 
+- Updates to the OID registry by associating the registry with Sweden Connect technical framework, fixing registered module OID:s and adding a new Extension OID for Signature Validation Tokens in timestamp extensions.
```
Comment (martin-lindstrom) [inline ELN-0603 - Registry for Identifiers.md]: This is not a Git commit comment. Make one bullet per relevant change.

File: ELN-0603 - Registry for Identifiers.md (hunk 2 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: e-legitimationsnamnden(201) is a registered name: see: http://oid-info.com/get/1.2.752.201

File: ELN-0603 - Registry for Identifiers.md (hunk 4 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: No, This is an accident

File: ELN-0603 - Registry for Identifiers.md (hunk 5 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: OK

File: ELN-0603 - Registry for Identifiers.md (hunk 3 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: Fixed

File: ELN-0603 - Registry for Identifiers.md (hunk 4 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: Added it back into this PR

File: ELN-0603 - Registry for Identifiers.md (hunk 6 above)
Comment (Razumain) [reply ELN-0603 - Registry for Identifiers.md]: Fixed
